File: mcp/jcodemunch-mcp/tests/fixtures/java/Sample.java
package sample;

/**
 * User service
 */
public class Sample {
    public static final int MAX_RETRIES = 3;

    /**
     * Get user by ID
     */
    public String getUser(int userId) {
        return "user-" + userId;
    }

    public static boolean authenticate(String token) {
        return token != null && !token.isEmpty();
    }
}

File: mcp/jcodemunch-mcp/tests/fixtures/dart/sample.dart
/// User service for managing users.
class UserService {
  /// Get user by ID.
  String getUser(int userId) {
    return 'user-$userId';
  }

  /// Delete a user.
  bool deleteUser(int userId) {
    return true;
  }

  /// Whether the service is ready.
  bool get isReady => true;
}

/// Scrollable behavior.
mixin Scrollable on UserService {
  /// Scroll to offset.
  void scrollTo(double offset) {}
}

/// Authenticate a token.
bool authenticate(String token) {
  return token.isNotEmpty;
}

/// Status of a request.
enum Status { pending, active, done }

/// Helpers for String manipulation.
extension StringExt on String {
  /// Whether the string is blank.
  bool get isBlank => trim().isEmpty;
}

/// JSON map alias.
typedef JsonMap = Map<String, dynamic>;
